src/middleware/auth.middleware.ts

```diff
@@ -0,0 +1,31 @@
+import ApplicationLogger from 'ch-logging/lib/ApplicationLogger';
+import { NextFunction, Request, RequestHandler, Response } from 'express'
+import { inject } from 'inversify'
+import { provide } from 'inversify-binding-decorators'
+import { BaseMiddleware } from 'inversify-express-utils'
+import { authMiddleware, AuthOptions } from 'web-security-node'
+
+import { SEARCH_COMPANY_URI } from 'app/paths'
+import { getEnvOrDefault } from 'app/utils/env.util'
+import { newUriFactory } from 'app/utils/uri.factory'
+
+@provide(AuthMiddleware)
+export class AuthMiddleware extends BaseMiddleware {
```

Comment:
Yeah rather than testing this via a controller, we should have a dedicated test class for this. If you mock out the npm authMiddleware function, then this file's test becomes:
```
authMiddleware.handle(req, res, next)

verify(commonMiddlware(req, res, next)).called()
```
You only test the code you wrote. You just verify that you called the library. No need to retest the library.